channelserver/src/server.rs

```diff
@@ -125,8 +125,8 @@ impl Default for ChannelServer {
             sessions: HashMap::new(),
             rng: RefCell::new(rand::thread_rng()),
             log: logger.clone(),
-            settings: RefCell::new(settings.clone()),
-            metrics: RefCell::new(metrics.clone())
+            settings: RefCell::new(settings),
```

Comment:
settings is immutable so it doesn't need a RefCell wrapper.

Despite some of actix's other examples, I don't see why any basic state within  actix::Handler implementors (Actors) need any interior mutability because actix::Handler::handle takes a &mut self

which is totally different from actix_web's **_Request Handlers_** (actix_web::dev::Handler::handle takes &self instead and requires interior mutability)

Reply:
Heh, so part of that was due to what I believed was a lock race condition showing up in my testing. 

No, it was just a bad test. 

Thanks!